# Define the goal  定义目标信息
uint32 dishwasher_id # Specify which dishwasher we want to use
---
# Define the result  定义结果信息
uint32 total_dishes_cleaned
---
# Define a feedback message 定义周期反馈的信息
float32 percent_complete